Source/Analytics/Core/Overview/LastWeekTotals/LastWeekTotalsController.cs

```diff
@@ -0,0 +1,41 @@
+
+using System.Security.Claims;
+using Concepts;
+using Dolittle.AspNetCore.Execution;
+using Dolittle.DependencyInversion;
+using Dolittle.Execution;
+using Dolittle.ReadModels;
+using Dolittle.Security;
+using Dolittle.Tenancy;
+using Microsoft.AspNetCore.Mvc;
+using Read.Overview.LastWeekTotals;
+
+namespace Core.Overview.LastWeekTotals
+{
+    [Route("api/CaseReport/Totals")]
```

Comment:
Is it implicitly understandable what kind of totals this is (comparing with TotalsPerHealthRisk)? Is it total cases for a project?

Reply:
We kept the URL that was used in Analytics before, to not change the frontend that was there before. So yes, that can definitely be discussed. But maybe outside of this PR? ☺️

Reply:
Yes, let's discuss that outside of this PR.